### PR TITLE
Fix/redesign list view

### PR DESCRIPTION
**Pull request by martinegeli**
- Easy design-change
- Redirects to detailview
- Removed unused imports

### CODE REVIEW

File: src/containers/UserPage/ListView.js [hunk 1]
```diff
@@ -1,52 +1,89 @@
...
 
+        if (events.length === 0) {
+            return (
+                <div>
+                    There is no events
+                </div>
+            )
+        }
+
+        let closestEvent = events.sort((a,b) => a.starttime - b.starttime)[0];
```
Comment (ankile) [inline src/containers/UserPage/ListView.js]: The events should be ordered from the database, so this sorting is probably unnecessary.

**Review by ankile — CHANGES_REQUESTED** [review]
Looks nice!
One comment in the code that probably should be changed to increase performance. Also, prettier should be enabled by default or run cmd + alt + shift + p to format the files before commit.

File: src/containers/UserPage/ListView.js (hunk 1 above)
Comment (martinegeli) [reply src/containers/UserPage/ListView.js]: Should be fixed now.

**Review by ankile — APPROVED** [review]
Nice man, the world is probably not ready for this revolution in list-view technology, but someone has to lead.

**Pull request merged**